spec/requests/categories_spec.rb

```diff
@@ -8,18 +8,17 @@
 #     end
 #   end
 
+# describe 'GET /show' do
+#   it 'returns http success' do
+#     get '/categories/show'
+#     expect(response).to have_http_status(:success)
+#   end
+# end
 
-  # describe 'GET /show' do
-  #   it 'returns http success' do
-  #     get '/categories/show'
-  #     expect(response).to have_http_status(:success)
-  #   end
-  # end
-
-  # describe 'GET /new' do
-  #   it 'returns http success' do
-  #     get '/categories/new'
-  #     expect(response).to have_http_status(:success)
-  #   end
-  # end
+# describe 'GET /new' do
+#   it 'returns http success' do
+#     get '/categories/new'
+#     expect(response).to have_http_status(:success)
+#   end
+# end
```

Comment:
- [ ] Kindly uncomment the tests' blocks of code and refactor them to accept test cases according to the project requirements to have units and integration tests. Please do so in all of the other related files! 🙏

Reply:
I'll do that!